src/renderer/modules/webpack/index.ts

```diff
@@ -737,6 +564,48 @@ export function getByProps<
   return getExportsForProps<P, T & ModuleExportsWithProps<P>>(result as T & ModuleExports, props);
 }
 
+export function waitForProps<
+  P extends string = string,
+  T extends ModuleExportsWithProps<P> = ModuleExportsWithProps<P>,
+>(props: P[], options: WaitForOptions & { raw?: false }): Promise<T>;
+export function waitForProps<P extends string = string, T extends RawModule = RawModule>(
+  props: P[],
+  options: WaitForOptions & { raw?: true },
+): Promise<T>;
+export function waitForProps<
+  P extends string = string,
+  T extends ModuleExportsWithProps<P> | RawModule = ModuleExportsWithProps<P> | RawModule,
+>(props: P[], options?: WaitForOptions): Promise<T>;
+export function waitForProps<
+  P extends string = string,
+  T extends ModuleExportsWithProps<P> = ModuleExportsWithProps<P>,
+>(...props: P[]): T;
+
+/**
```

Comment:
deletes all tsdoc... adds two docs anyway

Reply:
It doesn't delete them _all_, it just moves them to the actual function instead of the overloads. This is intentional.